blob/driver/driver.go

```diff
@@ -61,6 +61,9 @@ type WriterOptions struct {
 	// write in a single request, if supported. Larger objects will be split into
 	// multiple requests.
 	BufferSize int
+	// MaxUploadParts specifies the maximum number of uploaded parts.
```

Comment:
Let's model this comment after the previous one:


// MaxUploadParts changes the maximum number of uploaded parts, if supported.

No need to mention the S3 default.